src/main/resources/hudson/scm/SubversionSCM/excludedRevprop-validation.js

```diff
@@ -0,0 +1,31 @@
+Behaviour.specify("input[name='_.remote']", 'SubversionSCM.RemoteLocation', 0, function(element) {
+    element.addEventListener('blur', updateHiddenFields);
+});
+
+Behaviour.specify("select[name='_.credentialsId'][filldependson='remote']", 'SubversionSCM.CredentialsId', 0, function(element) {
+    element.addEventListener('change', updateHiddenFields);
+});
+
+function updateHiddenFields() {
+
+    var remoteLocationElement = document.querySelector("input[name='_.remote']");
+    var credentialsIdElement = document.querySelector("select[name='_.credentialsId'][filldependson='remote']");
+    var selectedOption = credentialsIdElement.options[credentialsIdElement.selectedIndex].value;
+
+
+    var remoteHidden = document.querySelector("input[name='_.remoteLocation']");
+    var credentialsHidden = document.querySelector("input[name='_.remoteCredentialsId']");
```

Comment:
I'd add classes to those elements and use that as selector instead.
Unlikely, but I could imagine fields with such names being added to the class, which will likely break this part of the code.